generator/simplecpp/simplecpp.h

```diff
@@ -39,6 +39,9 @@
 #endif
 
 namespace simplecpp {
+
+    std::string get_apple_framework_relative_path(const std::string& header);
+
```

Comment:
Does this have to be public, or could you just put it into an anonymous namespace in the implementation?